<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneCorrectDlg</class>
 <widget class="QDialog" name="PhoneCorrectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>216</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>348</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>348</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>348</width>
     <height>32</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="logo_label">
        <property name="text">
         <string>纠错配置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>32</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>40</y>
     <width>346</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>131</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>131</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="gn_ng_Check">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>gn -&gt; ng</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>如: lign -&gt; ling</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="mg_ng_Check">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>mg -&gt; ng</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>如: homg -&gt; hong</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="iou_iu_Check">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>iou -&gt; iu</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>如: qiou -&gt; qiu</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="uei_ui_Check">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>uei -&gt; ui</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>如: tuei -&gt; tui</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="uen_un_Check">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>uen -&gt; un</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>如: kuen -&gt; kun</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>175</y>
     <width>346</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="resetBtn">
      <property name="minimumSize">
       <size>
        <width>106</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>106</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>恢复默认设置</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="confirmBtn">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>85</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
